snap/local/runtime-helpers/bin/kong-daemon.sh

```diff
@@ -11,5 +11,8 @@ until "$KONG" migrations bootstrap --conf "$KONG_CONF"; do
     sleep 5
 done
 
+# set up Kong's admin API plugins via kong.yml:
+"$KONG" config db_import "$KONGADMIN_CONFIGFILEPATH"
```

Comment:
it should, but i think i can add `|| true` at the end of this command just in case then

Reply:
The "kong config db_import <file>" command does not return a clean exit code even though it successfully imports. It returns like "6" or something like that - the LUA code behind the scenes is where I found that. 

To get around that, so that the script will continue, I added the `|| true` to the end of the command, and it carries through gracefully. If the import fails, it will continue as if it were good...however, if the import does fail, there are going to be failures immediately down the line when proxy-setup attempts to start - so it doesn't concern me that much because if it does a fail/continue/fail because it won't end up in an exposed middle state - it will fail to setup the Admin API completely and proxy-setup won't be able to complete successfully.

Reply:
added `|| true`